oauth2-server/src/main/java/com/clouway/oauth2/jws/Signature.java

```diff
@@ -12,10 +17,14 @@
    * Verify is verifying Signature using the provided privateKey as PEM file.
    * <p/>
    *
-   * @param content the content to be verified
+   * @param content    the content to be verified
    * @param privateKey the private key used for verifying
    * @return true if signature is
    */
   boolean verify(byte[] content, Pem.Block privateKey);
 
+  boolean verifyWithPublicKey(byte[] content, Block key);
```

Comment:
As verification with private key (verify method) is specific it would be better: 

verify() to be renamed to: verifyWithPrivateKey 
verifyWithPublicKey to be renamed to just verify()

So the method signatures will look as follow: 

```java
// verify with public key
verify(byte[] content, Pem.Block publicKey);

// verify with private key
verifyWithPrivateKey(byte[] content, Pem.Block privateKey);

// sign with private key
sign(byte[] content, Pem.Block privateKey);

```

Comments are missing for the new methods.